doc/dev/index.rst

```diff
@@ -171,11 +171,11 @@ Development-mode cluster
 You can start a development-mode Ceph cluster, after compiling the source,
 with::
 
-    cd src
-    install -d -m0755 out dev/osd0
-    ./vstart.sh -n -x -l
+    cd build
```

Comment:
maybe we can drop this section "Building from source" as a whole, as https://github.com/ceph/ceph/blob/master/doc/dev/quick_guide.rst#running-a-development-deployment also documents this area.